### DISCUSSION TITLE
ScrollRestoration with sub route

### DISCUSSION THREAD

**Discussion by raskyer** [remix-run/remix#2364] [Q&A]
Hello, I'm trying to achive this : https://github.com/remix-run/remix/tree/main/examples/route-modal

So I have the routes : 
- users/$id.tsx
- users/$id/feedbacks.$feedback.tsx

The feedbacks of the users are displayed in a table at the end of the page '/users/$id' and when I click on View I want to display a modal (that contain the feedback) so I have a link that point to /users/$id/feedbacks/$feedback.

But when the redirect occur, my scroll position go to the top of the page. Is it possible to remain on the scroll position of /users/$id ?

I try to add "replace" to the Link component but didn't work out.

Thanks :)

**Comment by lili21** [comment]
I guess you need to achieve this by yourself. 

https://github.com/remix-run/remix/blob/40a7a390063836607c76aad9754b7881f8c82303/packages/remix-react/scroll-restoration.tsx#L126

you can change this to `window.scrollTo(0, window.scrollY)`.

**Reply by lili21** [reply]
or maybe build your own ScrollRestoration component.

**Comment by lili21** [comment]
or maybe you could just remove the component, It should work
